woudc_data_registry/epicentre/instrument.py

```diff
@@ -178,6 +193,8 @@ def update(ctx, identifier, station, dataset, name, model, serial, geometry):
         instrument_['station_id'] = station
     if dataset:
         instrument_['dataset_id'] = dataset
+    if station and contributor:
+        instrument_['deployment_id'] = ':'.join([station, contributor])
```

Comment:
Something to think about: deployment ID should be updated if only one of the `station` or `contributor` is changed.

If you look at the `update_metadata` method used further below, it does something similar to update station name ID whenever either station name or station ID is changed. You could do something like that inside `epicenter/metadata` to for this issue.